canonical-web-and-design/ubuntu.com
# 8756 make account name optional

⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️
### BEFORE MERGING / AFTER QA
Contract API url and Stripe public key must be updated to production values. They can be found [here](https://docs.google.com/document/d/1BG2mRWUF9pHAiwLPgm-7gelrZdstjlkQit84kXBum4A), and also here:
- contract API url: https://contracts.canonical.com
- Stripe public key: pk_live_68aXqowUeX574aGsVck8eiIE

⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️


## Done

- Made account name optional in the payment modal
- Default to user name if there's no account name provided

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage/subscribe
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- try to fill out the address form without an account name


## Issue / Card

Fixes #8756
